prime-router/build.gradle.kts

```diff
@@ -26,23 +26,23 @@ import java.io.FileInputStream
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 import java.util.Properties
-import kotlin.collections.mutableMapOf
 
 plugins {
-    kotlin("jvm") version "1.7.0"
+    kotlin("jvm") version "1.6.21"
     id("org.flywaydb.flyway") version "8.5.13"
```

Comment:
Why do you have to downgrade the JVM from 1.7 to 1.6?  We should try to use the latest/new stable version, isn't it?